courses/free-courses-ar.md

```diff
@@ -55,11 +55,13 @@
 
 ### Assembly
 
+* [Microprocessor 8086 & Assembly Language Course](https://www.youtube.com/playlist?list=PLi0-RQZxQ8Fmwopq43StX61igOvXbFMQv) - Sherif Ezzat
```

Comment:
```suggestion
* [Microprocessor 8086 & Assembly Language Course&rlm;](https://www.youtube.com/playlist?list=PLi0-RQZxQ8Fmwopq43StX61igOvXbFMQv) - Sherif Ezzat
```

Reply:
because the name of playlist is English not  Arabic